mobile/src/bundles/common/components/input/input.tsx

```diff
@@ -10,39 +10,69 @@ import { TextInput } from 'react-native';
 import { useFormController } from '~/bundles/common/hooks/hooks';
 import { globalStyles } from '~/bundles/common/styles/global-styles';
 
+import { Text, View } from '../components';
 import { styles } from './styles';
 
 type Properties<T extends FieldValues> = TextInputProps & {
     control: Control<T, null>;
     name: FieldPath<T>;
     hasError?: boolean;
+    marker?: string;
+    inputType?: 'text' | 'number';
+    multilines?: boolean;
```

Comment:
Some of this should already be in TextInputProps

Reply:
Oh, yes, we have 'multiline' already.